src/Nox.Generator/Application/Factories/EntityLocalizedFactory.template.cs

```diff
@@ -18,36 +18,52 @@ internal partial class {{className}} : {{className}}Base
 
 internal abstract class {{className}}Base : IEntityLocalizedFactory<{{localizedEntityName}}, {{entity.Name}}Entity, {{entity.Name}}UpdateDto>
 {
-    public virtual {{localizedEntityName}} CreateLocalizedEntity({{entity.Name}}Entity entity, CultureCode cultureCode)
+    public virtual {{localizedEntityName}} CreateLocalizedEntity({{entity.Name}}Entity entity, CultureCode cultureCode, bool withAttributes = true)
```

Comment:
For better readability:

```suggestion
    public virtual {{localizedEntityName}} CreateLocalizedEntity({{entity.Name}}Entity entity, CultureCode cultureCode, bool copyEntityAttributes= true)
```